hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -919,13 +919,13 @@ public Response pullRailsPortToDb(@Context HttpServletRequest request,
             // Move the data to the folder
             InternalCommand setFolder = updateParentCommandFactory.build(jobId, parentFolderId, layerName, params.getUser(), this.getClass());
             workflow.add(setFolder);
-        }
 
-        // Clean up pulled files
-        ArrayList<File> deleteFiles = new ArrayList<>();
-        deleteFiles.add(params.getWorkDir());
-        InternalCommand cleanFolders = removeFilesCommandFactory.build(jobId, deleteFiles);
-        workflow.add(cleanFolders);
+            // Clean up pulled files
+            ArrayList<File> deleteFiles = new ArrayList<>();
+            deleteFiles.add(params.getWorkDir());
+            InternalCommand cleanFolders = removeFilesCommandFactory.build(jobId, deleteFiles);
+            workflow.add(cleanFolders);
+        }
```

Comment:
So this effectively only deletes the pulled osm files if they are ingested into hoot api db.
I assume those that are not ingested into the hoot db are being used in one of the bulk tasks.  How do the files get cleaned up when the bulk task completes?

Reply:
They currently don't and I should add a call to remove the files in the workflow for the bulk jobs

Reply:
Ok, so you will add that and then we'll merge this?

Reply:
yup. on it